src/contacts.cs

```diff
@@ -545,6 +550,33 @@ interface CNContainer : NSCopying, NSSecureCoding {
 
 		[Export ("type", ArgumentSemantic.Assign)]
 		CNContainerType ContainerType { get; }
+
+#region comes from CNContainer (Predicates) Category
+		[Static]
+#if XAMCORE_4_0
+		[Export ("predicateForContainersWithIdentifiers:")]
+#else
+		[Wrap ("(null as CNContainer).GetPredicateForContainers (identifiers)")]
+#endif
+		NSPredicate CreatePredicateForContainers (string [] identifiers);
+
+		[NoiOS][NoWatch]
```

Comment:
`[NoiOS][NoWatch]` doesn't match what's in `CNContainer_PredicatesExtension`, is this correct?